audio/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -0,0 +1,102 @@
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/audioBackground"
+    android:paddingStart="16dp"
+    android:paddingEnd="16dp"
+    android:paddingBottom="26dp">
+
+    <TextView
+        android:id="@id/exo_position"
+        style="@style/MetadataMediumGray"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ideal_sans_book"
+        android:includeFontPadding="false"
+        android:paddingEnd="2dp"
+        android:textSize="14sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="0:15" />
+
+    <TextView
+        android:id="@+id/divider"
+        style="@style/MetadataMediumGray"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ideal_sans_book"
+        android:includeFontPadding="false"
+        android:text="/"
+        android:textSize="14sp"
+        app:layout_constraintStart_toEndOf="@id/exo_position"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <TextView
+        android:id="@id/exo_duration"
+        style="@style/MetadataMediumGray"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ideal_sans_book"
+        android:includeFontPadding="false"
+        android:paddingStart="2dp"
+        android:paddingRight="4dp"
+        android:textColor="#FFBEBEBE"
+        android:textSize="14sp"
+        app:layout_constraintStart_toEndOf="@id/divider"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="1:30" />
+
+
+    <FrameLayout
+        android:id="@+id/playPause"
+        android:layout_width="42dp"
+        android:layout_height="42dp"
+        android:layout_marginTop="20dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/exo_position">
+
+        <ImageButton
+            android:id="@id/exo_play"
+            android:layout_width="42dp"
+            android:layout_height="42dp"
+            android:background="@drawable/ic_play" />
+
+        <ImageButton
+            android:id="@id/exo_pause"
+            android:layout_width="42dp"
+            android:layout_height="42dp"
+            android:background="@drawable/ic_pause" />
+    </FrameLayout>
+
+    <com.google.android.exoplayer2.ui.DefaultTimeBar
+        android:id="@id/exo_progress"
+        android:layout_width="0dp"
+        android:layout_height="42dp"
+        android:layout_marginTop="20dp"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp"
+        app:bar_height="2dp"
+        app:buffered_color="#868686"
+        app:layout_constraintStart_toEndOf="@id/playPause"
+        app:layout_constraintTop_toBottomOf="@id/exo_position"
+        app:played_color="#37a2a5"
```

Comment:
no hardcoded hex. you need to reference a color style for clarity.

Reply:
updated.